test/DynamoCoreWpfTests/DynamoCoreWpfTests.csproj

```diff
@@ -1,114 +1,40 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <ImportGroup Label="PropertySheets">
-    <Import Project="$(SolutionDir)Config/CS.props" />
-    <Import Project="$(SolutionDir)Config/Cleanup.targets" />
```

Comment:
is this still relevant?

Reply:
not relevant anymore.
No 3d parties in nodes
and XMLs of 3d parties are not copied over by default